docs/guides/advanced/class-transformer.md

```diff
@@ -3,7 +3,7 @@ id: using-with-class-transformer
 title: 'Using with class-transformer'
 ---
 
-This guide shows how to use `typegoose` with `class-transformer`.
+This guide shows how to use `Typegoose` with `class-transformer`.
```

Comment:
it is an inline-code-block to show package names

Reply:
I got carried away I guess.

Scott